lua/cfc_notifications/client/cl_render.lua

```diff
@@ -59,14 +59,19 @@ hook.Add( "PlayerButtonDown", "CFCNotifications_render_keydown", function( ply,
     local x, _ = input.GetCursorPos()
     local wide = CFCNotifications.getSetting( "size_x" )
 
-    if key == KEY_F3 or ( clickerEnabled and key == MOUSE_LEFT and x < ( ScrW() - wide ) ) then
+    local shouldToggle = key == KEY_F3 or ( clickerEnabled and key == MOUSE_LEFT and x < ( ScrW() - wide ) )
```

Comment:
I think everything after the `or` should be split into it own descriptive variable so that `shouldToggle` reads very easily without needing to parse the `or`

Reply:
Fiiiiiiiiiiiiiiiiiiiiiiine